applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.h

```diff
@@ -0,0 +1,317 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Riccardo Tosi
+//
+
+#ifndef KRATOS_SYMBOLIC_D_CONVECTION_DIFFUSION_EXPLICIT_H
+#define KRATOS_SYMBOLIC_D_CONVECTION_DIFFUSION_EXPLICIT_H
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "symbolic_qs_convection_diffusion_explicit.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class SymbolicDConvectionDiffusionExplicit
+ * @ingroup ConvectionDiffusionApplication
+ * @brief This element solves the convection-diffusion equation, stabilized with
+ * algebraic subgrid scale or orthogonal subgrid scale.
+ * @details This element solves the convection-diffusion equation:
+ * $ \frac{\partial \phi}{\partial t} + v \cdot  \nabla \phi + \phi \nabla \cdot v - \nabla \cdot k \nabla \phi = f $
+ * where $ \phi $ is the scalar unknown, $ v $ the convective velocity, $ k > 0 $ the diffusivity coefficient,
+ * $ f $ the forcing term.
+ * Dynamic algebraic subgrid scale and dynamic orthogonal subgrid scale methods are exploited for stabilization.
+ * The element is designed to use an explicit integration method.
+ * @author Riccardo Tosi
+ */
+template< unsigned int TDim, unsigned int TNumNodes = TDim + 1>
```

Comment:
```suggestion
template< unsigned int TDim, unsigned int TNumNodes>
```
I'd avoid defaults of this type in order to always make clear which geometry is being specialized.

Reply:
Ok. I guess we made this decision together with @RiccardoRossi, but I have nothing against your suggestion. Therefore, I will apply it.

Reply:
Thanks. I think it makes thing more clear.